migrations/Version202608051012062141_taoItems.php

```
@@ -0,0 +1,88 @@
+            self::GLOBAL_MANAGER_ROLE => [
+                self::REST_ITEM_COMMENTS_MASK,
+            ],
+        ],
+    ];
+
+    public function getDescription(): string
+    {
+        return 'Grant RestItemComments ACL for authoring roles (NYSED-13)';
+    }
+
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)